docs/api/web-frame.md

```diff
@@ -221,4 +221,13 @@ in your app has occurred that makes you think your page is actually using less
 memory (i.e. you have navigated from a super heavy page to a mostly empty one,
 and intend to stay there).
 
+### `webFrame.setCacheCapacity(capacity)`
+
+* `capacity` Integer
+
+Attempts to set maximum size of memory cache web frame uses.
```

Comment:
Can you please mention the units of the `capacity` here? Is it bytes, MBs, something else?

Reply:
yeah, good catch. updated it. Internally Chromium seems have different size of caches per usage - for example, in case of MemoryCache it has 8MB (`kCDefaultCacheCapacity = 8192 * 1024`) as the default size in ctor.